src/averageAges.js

```diff
@@ -15,13 +15,28 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = century
+    ? people.filter(person => century === Math.ceil(person.died / 100)
+      && person.sex === 'm')
+    : people.filter(person => person.sex === 'm');
```

Comment:
```suggestion
  const isMale = target => target.sex === 'm';
  const men = century
    ? people.filter(person => century === Math.ceil(person.died / 100)
      && isMale(person)
    : people.filter(person => isMale(person));
```
Just for repeating code and ternary readability imo (also for consts `woman `& `children`)